examples/basic-tcp.yaml

```diff
@@ -37,5 +42,18 @@ metadata:
 spec:
   rules:
   - forwardTo:
-    - serviceName: my-service
+    - serviceName: my-foo-service
+      port: 6000
+---
+kind: TCPRoute
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: tcp-app-2
+  namespace: default
```

Comment:
I think we should probably remove any hardcoded namespace references throughout our examples. Leaving these in makes it harder to deploy an example in a test namespace.

Reply:
I had this exact same thought but left it in because other examples had it.
Tracking this in https://github.com/kubernetes-sigs/service-apis/issues/498 to avoid scope creep here.